packages/tangle/scheduler.go

```diff
@@ -122,15 +126,32 @@ func (s *Scheduler) Setup() {
 			return
 		}
 
-		if err := s.SubmitAndReady(messageID); err != nil {
+		if err := s.Submit(messageID); err != nil {
 			if !errors.Is(err, schedulerutils.ErrBufferFull) &&
 				!errors.Is(err, schedulerutils.ErrInboxExceeded) &&
 				!errors.Is(err, schedulerutils.ErrInsufficientMana) {
```

Comment:
AFAIK those are already logged when they're thrown. However, I removed most of those in a separate PR, as currently the only possible error will be `schedulerutils.ErrInsufficientMana` and it will be logged immediately.